### PR TITLE
Tidying up sources

### PR DESCRIPTION
**Pull request by JoshyPHP**
Just tidying up sources. No functional change intended.

### CODE REVIEW

File: core/parser.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function replace_mentions_for_display($text)
 			$start_text = $text;
 		}
 
-		for($i; $i < 100; $i++)
+		for ($i; $i < 100; $i++)
```
Comment (Wolfsblvt) [inline core/parser.php:69]: @s9e This code will be gone in any beta version anyway, it was just quick'n'dirty coded for time measuring and I kept it for testing purposes.
But you are right indeed. I'll keep that until I will remove that code.

File: event/listener.php [hunk 2]
```diff
@@ -78,7 +78,6 @@ static public function getSubscribedEvents()
 
 			// Message parsing
 			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
-			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
```
Comment (Wolfsblvt) [inline event/listener.php:81]: @s9e Oh, this is the same event indeed, ye. But I was quite sure I need another one here.
Should be `core.modify_text_for_display_before`, shouldn't it?
I think I've seen bot events in use by advanced bbcode extension.

When this part is clear, I'll merge that pull request.

Comment (JoshyPHP) [inline event/listener.php:81]: `core.modify_text_for_display_before` is the one that's already in there. You might be thinking of `core.modify_format_display_text_after`, although I'm not sure it applies to your extension.

Comment (Wolfsblvt) [inline event/listener.php:81]: Uh what. I am really too tired today to work on extensions ._.
I think I meant that, yes. There are two events. I'll test that out, but you are right I would say.

**Pull request merged**